src/App.tsx

```diff
@@ -1,10 +1,55 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoInput } from './components/TodoInput/TodoInput';
+import { FilterType } from './types/FilterType';
+import { Notification } from './components/Notification/Notification';
 
-const USER_ID = 0;
+const USER_ID = 7022;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState<FilterType>(FilterType.All);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(data => setTodos(data))
+      .catch(() => {
+        setIsError(true);
+        setTimeout(() => {
+          setIsError(false);
+        }, 3000);
+      });
+  }, [todos]);
+
+  const activeTodos = useMemo(
+    () => todos.filter(todo => !todo.completed), [todos],
+  );
+
+  const completedTodos = useMemo(
+    () => todos.filter(todo => todo.completed), [todos],
+  );
+
+  const filteredTodos = useMemo(() => {
+    switch (filterType) {
+      case FilterType.Completed:
+        return todos.filter(todo => todo.completed);
+
+      case FilterType.Active:
+        return todos.filter(todo => !todo.completed);
+
+      default:
+        return todos;
+    }
+  }, [todos, filterType]);
+
+  const handleChangeFilter = (filter: FilterType) => {
+    setFilterType(filter);
+  };
```

Comment:
Change handler this way and use how I show below
```suggestion
  const handleChangeFilter = (filter: FilterType) => () => {
    setFilterType(filter);
  };
```